pyproject.toml

```
@@ -12,7 +12,7 @@ exclude = [
 homepage = "https://ibis-project.org"
 repository = "https://github.com/ibis-project/ibis"
 documentation = "https://ibis-project.org"
-description = "Productivity-centric Python Big Data Framework"
+description = "the portable Python dataframe library"
```

I think we should capitalize `The`.

```suggestion
description = "The portable Python dataframe library"
```